lib/ansible/modules/database/misc/kibana_saved_object.py

```diff
@@ -0,0 +1,446 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2019, Ruben Tsirunyan @rubentsirunyan
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: kibana_saved_object
+short_description: Manage Kibana saved objects
+description:
+    - This module can be used to Create/Update/Delete Kibana saved objects.
+version_added: "2.8"
+author: Ruben Tsirunyan (@rubentsirunyan)
+options:
+    id:
+      type: str
+      description:
+        - ID of the object.
+      required: True
+    state:
+      type: str
+      description:
+        - Desired state of the object.
+      choices: ["present", "absent"]
+      default: present
+    kibana_url:
+      type: str
+      description:
+        - The URL of Kibana.
+    user:
+      type: str
+      description:
+        - A username for the module to use for Digest, Basic or WSSE authentication.
+      aliases: [url_username]
+    password:
+      type: str
+      description:
+        - A password for the module to use for Digest, Basic or WSSE authentication.
+      aliases: [url_password]
+    force_basic_auth:
+      type: bool
+      description:
+        - The library used by the module only sends authentication information when a webservice
+          responds to an initial request with a 401 status. Since some basic auth services do not properly
+          send a 401, logins will fail. This option forces the sending of the Basic authentication header
+          upon initial request.
+      default: false
+    use_proxy:
+      type: bool
+      description:
+        - if C(false), it will not use a proxy, even if one is defined in
+          an environment variable on the target hosts.
+      default: true
+    validate_certs:
+      description:
+        - If C(no), SSL certificates will not be validated.  This should only
```

Comment:
with `use_proxy` you use C(false), here you use C(no). It would look cleaner to use either C(false) everywhere or C(no).

Reply:
Done.